### DISCUSSION TITLE
第1节课笔记(19班)

### DISCUSSION THREAD

**Discussion by LYMDLUT** [InternLM/Tutorial#52] [笔记]
第一节课笔记（19班）

**Comment by WiselyC0430** [comment]
http://t.csdnimg.cn/pOIOE

**Comment by jts250** [comment]
https://zhuanlan.zhihu.com/p/675943932

**Comment by jia-heng** [comment]
https://github.com/jia-heng/puyu-learn/tree/main/lesson1

**Comment by Sixdes** [comment]
https://bbs.csdn.net/topics/617848281

**Comment by hu-qi** [comment]
[书生·浦语全链条开源开放体系详解](https://mp.weixin.qq.com/s?__biz=MzI2NjMwODY4MQ==&mid=2247487079&idx=1&sn=c084c583c8d17bdd40709c639c264722&chksm=ea915341dde6da57c9cb10b30025d8b912dee3691ba48adae3827dbbbb924e36ab72eaca14c8#rd)

**Comment by hellboyvto** [comment]
http://t.csdnimg.cn/LDAnn

**Comment by Jiangzzzzl** [comment]
https://blog.csdn.net/2301_80618119/article/details/135376409?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135376409%22%2C%22source%22%3A%222301_80618119%22%7D

**Comment by fusheng-ruomeng** [comment]
https://jerry-mao.notion.site/Lesson1-c299bcc12b7f4251a3819761dd9ec4aa?pvs=4

**Comment by JankinTian** [comment]
https://blog.csdn.net/xiao_xian_/article/details/135407055

**Comment by WeiLiuAH** [comment]
https://zhuanlan.zhihu.com/p/676296936

**Comment by Haru-zt** [comment]
https://zhuanlan.zhihu.com/p/675928910

**Comment by romanwin28** [comment]
https://blog.csdn.net/George_han/article/details/135392569

**Comment by haohaocodeli** [comment]
http://t.csdnimg.cn/Vn4Tb

**Comment by melonwine** [comment]
https://github.com/melonwine/InternLMPlay/blob/master/01/note01.md

**Comment by AllYoung** [comment]
https://blog.csdn.net/sunshine_youngforyou/article/details/135439124

**Comment by yunlong96** [comment]
https://blog.csdn.net/li_kin/article/details/135395916?spm=1001.2014.3001.5501

**Comment by SnowKingLiu** [comment]
https://github.com/SnowKingLiu/puyu/blob/main/course1/Course1.md

**Comment by yunlong96** [comment]
![image](https://github.com/InternLM/tutorial/assets/53212855/116cbf23-3da1-449a-a3de-d9e6c5786638)

**Comment by mkdirmushroom** [comment]
[第1节课笔记](https://github.com/mkdirmushroom/note/blob/main/InternLM/tutorial0_note.md)

**Comment by Hamasuta666** [comment]
第一节课笔记：https://zhuanlan.zhihu.com/p/680521466
